inst/Upgrades.qml

```diff
@@ -0,0 +1,171 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades
+{
+	Upgrade
+	{
+		functionName:		"NetworkAnalysis"
+		fromVersion:		"0.15"
+		toVersion:			"0.16.4"
+
+		ChangeJS
+		{
+			name: "estimator"
+			jsFunction: function(options)
+			{
+				switch (options["estimator"])
+				{
+					case "EBICglasso":			return "ebicGlasso";
+					case "IsingFit":			return "isingFit";
+					case "IsingSampler":		return "isingSampler";
+					// the other values did not change
+					default:					return options["estimator"];
+
+				}
+			}
+		}
+
+		ChangeRename	{	from:	"plotNetwork";						to:		"networkPlot"									}
+		ChangeRename	{	from:	"plotCentrality";					to:		"centralityPlot"								}
+		ChangeRename	{	from:	"plotClustering";					to:		"networkclusteringPlotPlot"						}
+
+		ChangeRename	{	from: 	"tableCentrality";					to:		"centralityTable"								}
+		ChangeRename	{	from: 	"tableClustering";					to:		"clusteringTable"								}
+		ChangeRename	{	from: 	"tableWeightsMatrix";				to:		"weightsMatrixTable"							}
+
+		ChangeRename	{	from:	"normalizeCentrality";				to:		"centralityNormalization"						}
+
+		ChangeRename	{	from: 	"mgmVariableTypeContinuous";		to:		"mgmContinuousVariables"						}
+		ChangeRename	{	from: 	"mgmVariableTypeCategorical";		to:		"mgmCategoricalVariables"						}
+		ChangeRename	{	from: 	"mgmVariableTypeCount";				to:		"mgmCountVariables"								}
+
+		ChangeRename	{	from:	"BootstrapType";					to:		"bootstrapType"									}
+		ChangeRename	{	from:	"bootstrapOnOff";					to:		"bootstrap"										}
+		ChangeRename	{	from:	"numberOfBootstraps";				to:		"bootstrapSamples"								}
+		ChangeRename	{	from:	"parallelBootstrap";				to:		"bootstrapParallel"								}
+
+
+		ChangeRename	{	from: 	"StatisticsEdges";					to:		"statisticsEdges"								}
+		ChangeRename	{	from: 	"StatisticsCentrality";				to:		"statisticsCentrality"							}
+
+		ChangeRename	{	from: 	"groupNames";						to:		"manualColorGroups"								}
+
+		ChangeJS
+		{
+			name: "manualColorGroups"
+			jsFunction: function(options)
+			{
+
+				return options["manualColorGroups"].map(item=>{
+					item["name"] = item["group"];
+					item["color"] = item["groupColors"];
```

Comment:
Still some issues with the upgrades here... file to test: [network_graphical-options_0.16.3.jasp.zip](https://github.com/jasp-stats/jaspNetwork/files/9280812/network_graphical-options_0.16.3.jasp.zip)